WebApp-your-API/MyOrg/Client/Pages/ToDoPages/ToDoListBase.cs

```diff
@@ -1,23 +1,24 @@
 ﻿using Microsoft.AspNetCore.Components;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Identity.Abstractions;
 using Microsoft.Identity.Web;
 using System;
 using System.Collections.Generic;
+using System.Net.Http;
 using System.Threading.Tasks;
 using ToDoListModel;
 
 namespace blazorserver_client.Pages.ToDoPages
 {
-
+   
     public class ToDoListBase : ComponentBase
     {
-        [Inject]
-        ToDoListService ToDoListService { get; set; }
 
         [Inject]
         MicrosoftIdentityConsentAndConditionalAccessHandler ConsentHandler { get; set; }
 
         [Inject]
-        NavigationManager Navigation { get; set; }
+        IDownstreamApi _downstreamApi { get; set; }
```

Comment:
It's a property, you might want to name it DownstreamApi?

Reply:
As it's currently defines, the partial class DownstreamApi is internal and inaccessible here